packages/dataviews/src/dataviews-layouts/table/style.scss

```diff
@@ -163,6 +163,18 @@
 			&:has(.dataviews-view-table-header-button):not(:first-child) {
 				padding-left: $grid-unit-05;
```

Comment:
If we just added `padding-right: $grid-unit-05;` here would that solve the issue without all the extra logic and CSS?